src/ComboBox/driver.js

```diff
@@ -0,0 +1,116 @@
+const ERR = {
+    DISABLED : () => 'Cannot trigger event on the disabled option'
+};
+
+export default class ComboBoxDriver
+{
+    constructor( wrapper )
+    {
+        this.wrapper = wrapper;
+    }
+
+    blur()
+    {
+        this.wrapper.find( 'InputField' ).driver().blur();
+        return this;
+    }
+
+    changeInput()
+    {
+        this.wrapper.find( 'InputField' ).driver().pressKey( 'c' );
+        return this;
+    }
+
+    clickInput()
+    {
+        this.wrapper.find( 'InputField' ).driver().click();
+        return this;
+    }
+
+    clickOption( index = 0 )
+    {
+        const option = this.wrapper.find( 'ListBoxOption' ).at( index );
+
+        if ( option.props().isDisabled )
+        {
+            throw new Error(
+                ERR.DISABLED()
+            );
+        }
+
+        option.simulate( 'click' );
+        return this;
+    }
+
+    focus()
+    {
+        this.wrapper.find( 'InputField' ).driver().focus();
+        return this;
+    }
+
+    keyDown()
+    {
+        this.wrapper.find( 'InputField' ).simulate( 'keyDown' );
+        return this;
+    }
+
+    keyPress()
```

Comment:
Should accept a string and pass it through to the `pressKey` driver method of the InputField